Java/core/src/main/java/com/amazon/randomcutforest/executor/PointStoreCoordinator.java

```diff
@@ -47,6 +48,6 @@ public void completeUpdate(List<Optional<UpdateReturn<Integer>>> updateResults,
     }
 
     public PointStoreDouble getStore() {
-        return store;
+        return (PointStoreDouble) store;
```

Comment:
We should update this method to return IPointStore and then update the corresponding mapper class to do the casting.

Reply:
Agree with the change - let's postpone changing the mapper in this PR?

Reply:
https://github.com/aws/random-cut-forest-by-aws/issues/78